src/WebJobs.Script.WebHost/App_Data/secrets/host.json

```diff
@@ -1,4 +1,4 @@
 ﻿{
-  "masterKey": "t8laajal0a1ajkgzoqlfv5gxr4ebhqozebw4qzdy",
-  "functionKey": "zlnu496ve212kk1p84ncrtdvmtpembduqp25ajjc"
+    "masterKey": "t8laajal0a1ajkgzoqlfv5gxr4ebhqozebw4qzdy,z3dlq50s00cb3q3k11nil7xyt29ebst2n8rtn0t3",
```

Comment:
I don't have a lot of experience rolling keys, but I would think it'd go something like a queue pattern:

- You normally have one key
- When you need to roll it, you add a second key at the end
- Once consumers have switched, you delete the one at the beginning

So it's very positional rather than name based. Is that not what a real world patter would look like?